write-config.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# expects these environment variables:
+#
+# - `ELASTICSEARCH_ENDPOINT`: URL to elasticsearch
+
+# Our index name
+INDEX=.kibana
+
+# Check env variable
+if [ "$ELASTICSEARCH_ENDPOINT" == "" ]; then
+  echo "ERROR: Environment variable ELASTICSEARCH_ENDPOINT is not set. Exiting."
+  exit 1
+fi
+
+while true
+do
+
+  # Wait for Elasticsearch to become available
+  OKAY="unset"
+  curl -s --head "$ELASTICSEARCH_ENDPOINT"|grep -q "HTTP/1.1 200"
+  OKAY=$?
+  while [ "$OKAY" != "0" ]; do
+    echo "Waiting for Elasticsearch to become available"
+    sleep 10
+    curl -s --head "$ELASTICSEARCH_ENDPOINT"|grep -q "HTTP/1.1 200"
+    OKAY=$?
+  done
+
+
+  # Check if index exists
+  curl -s --head "$ELASTICSEARCH_ENDPOINT/$INDEX"|grep -q "HTTP/1.1 200"
```

Comment:
I would do that for every creation request because, if during the 3 docs creation on failed, you will never catch it

Reply:
That's very unlikely, isn't it? And this operation is running hourly.

Reply:
À communication problem or timeout can happen, and if the first one was done but not the other 2 you will never enter in this loop anymore